draft-vvv-webtransport-quic.md

```diff
@@ -233,6 +233,42 @@ datagram ID SHALL be absent.
 
 The datagrams sent using QuicTransport MUST be subject to congestion control.
 
+# QuicTransport URI Scheme {#uri}
+
+NOTE: the URI scheme defintion in this section is provisional and subject to
+change, especially the name of the scheme.
+
+QuicTransport uses `quic-transport` URI scheme for identifying QuicTransport
+servers.
+
+The syntax definition below uses Augmented Backus-Naur Form (ABNF) {{!RFC5234}}.
+The definitions of `host`, `port`, `path-abempty`, `query` and `fragment` are
+adopted from {{!RFC3986}}.  The syntax of a QuicTransport URI SHALL be:
+
+~~~~~~~~~~~~~~~
+quic-transport-URI = "quic-transport:" "//"
+                             host [ ":" port ]
+                             [ "?" query ]
+                             [ "#" fragment ]
+~~~~~~~~~~~~~~~
+
+This document does not define any semantics to `path-abemtpy`, `query` or
+`fragment` portions of the URI, nor does it delegate those to the host owners.
+Any QuicTransport implementation MUST ignore those until a subsequent
+specification assigns semantics to those.
```

Comment:
add: and MUST NOT send non-empty values

Reply:
I'm not sure what you mean here.  We do not provide any way to send those in the protocol.

Reply:
Good point.

Reply:
Good point